src/AuditConfiguration.php

```diff
@@ -71,15 +71,15 @@ public static function forEntities(array $classes)
     }
 
     /**
-     * @param ClassMetadataInfo<object> $metadata
+     * @param ClassMetadata<object> $metadata
      *
      * @return string
      *
      * @phpstan-return literal-string
      *
      * @psalm-suppress MoreSpecificReturnType,LessSpecificReturnStatement https://github.com/vimeo/psalm/issues/10910
      */
-    public function getTableName(ClassMetadataInfo $metadata)
+    public function getTableName(ClassMetadata $metadata)
```

Comment:
As this class is not final or marked as internal, I think this is technically a BC break.
Maybe we can live with this documenting this decision.

Reply:
My bad (I was confusing `Doctrine\ORM\Mapping\ClassMetadata` with `Doctrine\Persistence\Mapping\ClassMetadata`).